examples/analytics-notification.amp.html

```diff
@@ -66,8 +66,8 @@
   <amp-user-notification
       layout=nodisplay
       id="amp-user-notification1"
-      data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
Does this actually work? I thought we might mandate `https` to be present.

There isn't really a great way to do this, I'm afraid, short of producing other example files (like we we do for `.min.`) We could do `.https.`.